starter-kits/product/_credits.md

```diff
@@ -1,4 +1,5 @@
 % ## Acknowledgments
+% - optional, but upstream data acknowledgments, where appropriate, are required
```

Comment:
Will your Metadata document already tell them that something is a 'required' or 'optional' field? I am wondering if we are duplicated content here.

Reply:
Agreed. I have removed the text, and just used the commented or not commented as the indicator.